OrangeHRMTests/PageObjects/BasePage.cs

```diff
@@ -0,0 +1,58 @@
+﻿using OpenQA.Selenium;
+using OrangeHRMTests.Common.WebElements;
+using OrangeHRMTests.PageObjects.Modules;
+
+namespace OrangeHRMTests.PageObjects
+{
+    public class BasePage
+    {
+        public LeftMenuNavigationPanel LeftMenuNavigationPanel => new LeftMenuNavigationPanel();
+
+        private static MyWebElement AddButton = new MyWebElement(By.XPath("//*[contains(@class, 'orangehrm-header-container')]//button"));
+        private static MyWebElement SaveButton = new MyWebElement(By.XPath("//button[@type='submit']"));
+        private static MyWebElement ConfirmDeletionButton = new MyWebElement(By.XPath("//*[contains(@class, 'orangehrm-modal')]//*[contains(@class, 'trash')]"));
+        private static MyWebElement SearchButton = new MyWebElement(By.XPath("//*[contains(@class, 'form')]//*[contains(@type, 'submit')]"));
+        private static MyWebElement DeleteSelectedButton = new MyWebElement(By.XPath("//*[contains(@class, 'padding')]//*[contains(@type, 'button')][contains(@class, 'danger')]"));
+        private static MyWebElement PencilEditButton = new MyWebElement(By.XPath("(//div[@class='oxd-table-card']//div[@role='row'])[1]//div[@role='cell'][count(//div[text()='Actions']//preceding-sibling::div)+1]//i[@class='oxd-icon bi-pencil-fill']"));
+        private static MyWebElement TrashButton = new MyWebElement(By.XPath("(//div[@class='oxd-table-card']//div[@role='row'])[1]//div[@role='cell'][count(//div[text()='Actions']//preceding-sibling::div)+1]//i[@class='oxd-icon bi-trash']"));
+        private static MyWebElement TableCheckboxElement = new MyWebElement(By.XPath("(//div[@class='oxd-table-card']//div[@role='row'])[1]//div[@role='cell']//i[@class='oxd-icon bi-check oxd-checkbox-input-icon']"));
```

Comment:
//div[@class='oxd-table-card']//div[@role='row'] appears in several elements

Reply:
Fixed